src/components/Sidebar/SectionSelection.js

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
 
-function SectionSelection({ currentCourseName, sections, back }) {
+export const styles = {
+  sectionsRoot: {
+    padding: 16,
+  },
+  title: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  divider: {
+    marginTop: 5,
+  },
+  sectionTitle: {
+    fontWeight: 'bold',
+  },
+};
+
+function SectionSelection({
+  currentCourseName,
+  sections,
+  back,
+  classes,
+}) {
   return (
-    <div>
-      <Typography variant="headline">{currentCourseName}</Typography>
-      <Button onClick={back}>Back</Button>
-      {sections.map(section => <div key={section.id}>{`Section ${section.section}`}</div>)}
+    <div className={classes.sectionsRoot}>
+      <Typography variant="headline" className={classes.title}>
+        {currentCourseName}
+        <Button onClick={back}>Back</Button>
+      </Typography>
+      <Divider className={classes.divider} />
+      <List>
+        {sections.map(section => (
+          <ListItem
+            key={section.id}
+            button
+          >
+            <ListItemText>
+              <Typography variant="h1" className={classes.sectionTitle}>{`Section ${section.section}`}</Typography>
+              <Typography>{`${section.class_mtg_info[0].meet_t}`}</Typography>
+              <Typography>{`${section.class_mtg_info[0].meet_l}`}</Typography>
+              <Typography>{section.instructor.map((t, i) => (`${t}${i !== section.instructor.length - 1 ? ', ' : ''}`))}</Typography>
```

Comment:
Let's try to avoid use of single letter variables names. We can definitely be descriptive AND concise.

Reply:
Also I feel like the length of this line should throw a linting error?

Reply:
Tests are also failing because we're missing coverage on this line